src/middlewares/__tests__/cors.js

```diff
@@ -0,0 +1,77 @@
+const middy = require('../../middy')
+const cors = require('../cors')
+
+describe('📦 Middleware CORS', () => {
+  test('It should default to "*"', () => {
```

Comment:
I would rename this test to something a bit more explicit like:

`Access-Control-Allow-Origin header should default to '*'`